a3_finance/overrides/salary_slip.py

```
@@ -1017,11 +1018,20 @@ def set_actual_amounts(doc, method):
             total += row.custom_actual_amount
 
-        else:
+        elif row.abbr == "Children Education Allowance":
             # For other components, either copy row.amount or set to 0
             row.custom_actual_amount = row.amount
             total += row.custom_actual_amount
 
+        elif row.abbr == "Washing Allowance":
+            row.custom_actual_amount = row.amount
+            total += row.custom_actual_amount
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)